.circleci/config.yml

```diff
@@ -77,30 +71,21 @@ commands:
           name: Install Rust
           command: |
             RUSTUP_PLATFORM=x86_64-apple-darwin
-            RUSTUP_VERSION=1.18.3
-            RUSTUP_SHA256=16734a9a2d87a3054bd4eea962642687e50a34b6e73e17df6f3361c2c534dc30
+            RUSTUP_VERSION=1.23.1
+            RUSTUP_SHA256=39101feb178a7e3e4443b09b36338e794a9e00385e5f44a2f7789aefb91354a9
             curl -sfSL --retry 5 --retry-delay 10 -O "https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${RUSTUP_PLATFORM}/rustup-init"
             echo "${RUSTUP_SHA256} *rustup-init" | shasum -a 256 -c -
             chmod +x rustup-init
             ./rustup-init -y --no-modify-path
             rm rustup-init
             echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> $BASH_ENV
   setup-rust-toolchain:
-    parameters:
-      rust-version:
-        type: string
-        default: << pipeline.parameters.stable-rust-version >>
     steps:
       - run:
           name: Set up Rust toolchain
           command: |
             rustup update
-            rustup install <<parameters.rust-version>>
-            rustup default <<parameters.rust-version>>
             rustc --version
-      - run:
-          name: Install UniFFI
-          command: cargo install --version 0.7.1 uniffi_bindgen
```

Comment:
This was leftover after a previous commit made it unnecessary, removing it while I'm here.